charts/opentelemetry-collector/values.yaml

```diff
@@ -285,6 +285,23 @@ ingress:
 #      hosts:
 #        - collector.example.com
 
+# Additional ingresses - only created if ingress.enabled is true
+# Useful for when differently annotated ingress services are required
+extraIngresses: []
+#  - name: cloudwatch
+#    ingressClassName: nginx
+#    annotations: {}
+#    hosts:
+#      - host: collector.example.com
+#        paths:
+#          - path: /
+#            pathType: Prefix
+#            port: 4318
+#    tls:
+#      - secretName: collector-tls
+#        hosts:
+#          - collector.example.com
```

Comment:
Does every other ingress needs a separate `tls` option or `ingress.tls` can be used instead?

Reply:
I would probably keep `tls` as different hosts could be configured in additional ingresses which require different TLS secrets? Could add an option in `additionalIngresses` to inherit TLS from the root ingress but maybe that's going too far.

Reply:
It depends on how common the use case when TLS is shared across all the ingresses. If it's not common at all let's not inherit it. Otherwise we may even omit the `tls` option in `additionalIngresses` until it's needed and use `ingress.tls` for now

Reply:
One of my specific use case just now uses seperate secrets per ingress (an AWS CloudWatch receiver with a standard public CA TLS certificate secret, and an OTLP receiver with a private CA TLS certificate secret used for client certificate authentication), so I'd prefer to just keep `tls` defined per additional ingress for now, and if there was feedback that a common TLS config was often being used across multiple ingresses by users, could add an `inheritTLS` option in `additionalIngresses` or something in the future?

Reply:
Ok, sounds good